idemp-rest/WEB-INF/src/org/bandahealth/idempiere/rest/model/InvoiceLine.java

```diff
@@ -31,9 +35,16 @@ public class InvoiceLine extends BaseMetadata {
 	public InvoiceLine() {
 	}
 
+	public InvoiceLine(MInvoiceLine instance) {
+		this(instance.getAD_Client_ID(), instance.getAD_Org_ID(), instance.getC_InvoiceLine_UU(), instance.isActive(),
```

Comment:
Can we just set the fields directly instead of using another constructor (since we want to eventually get rid of the other constructors)?

Reply:
ok